src/main/kotlin/com/example/noswimmingserver/domain/feedback/presentation/FeedbackController.kt

```diff
@@ -1,23 +1,41 @@
 package com.example.noswimmingserver.domain.feedback.presentation
 
 import com.example.noswimmingserver.domain.feedback.presentation.dto.request.CreateFeedbackRequest
-import com.example.noswimmingserver.domain.feedback.service.CreateFeedbackService
+import com.example.noswimmingserver.domain.feedback.presentation.dto.request.UpdateFeedbackRequest
+import com.example.noswimmingserver.domain.feedback.service.FeedbackService
 import org.springframework.http.HttpStatus
 import org.springframework.web.bind.annotation.*
 import javax.validation.Valid
 
 @RequestMapping("/feedback")
 @RestController
 class FeedbackController(
-    private val createFeedbackService: CreateFeedbackService,
+    private val feedbackService: FeedbackService,
 ) {
 
     @ResponseStatus(HttpStatus.CREATED)
     @PostMapping("/{reading-journal-id}")
     fun createFeedback(
         @PathVariable("reading-journal-id") readingJournalId: Long,
-        @RequestBody @Valid request: CreateFeedbackRequest
+        @RequestBody @Valid request: CreateFeedbackRequest,
     ) {
-        createFeedbackService.execute(readingJournalId, request)
+        feedbackService.execute(readingJournalId, request)
     }
-}
\ No newline at end of file
+
+    @ResponseStatus(HttpStatus.NO_CONTENT)
+    @PutMapping("/{feedback-id}")
+    fun editFeedback(
+        @PathVariable("feedback-id") feedbackId: Long,
+        @RequestBody @Valid request: UpdateFeedbackRequest,
```

Comment:
🚫 **[ktlint]** <annotation> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Multiple annotations should not be placed on the same line as the annotated construct